commands/uploader.go

```diff
@@ -327,7 +329,11 @@ func (c *uploadContext) Await() {
 	c.trackedLocksMu.Unlock()
 
 	if avoidPush {
-		Error("WARNING: The above files would have halted this push.")
+		if c.lockVerifyState == verifyStateEnabled {
+			Exit("ERROR: Cannot update locked files.")
+		} else {
+			Error("WARNING: The above files would have halted this push.")
```

Comment:
To make sure I'm understanding properly, does this table describe what you're thinking?

| `locksverify` | API request | verified | not verified | error |
|--------------|-------------|--------|-------------|------|
| `null` | yes | silent | warning | warning |
| `true` | yes | silent | error | error |
| `false` | yes | silent | warning | silent |